UnityProject/Assets/Scripts/Input System/InteractionV2/DefaultWillInteract.cs

```diff
@@ -43,6 +43,10 @@ public static class DefaultWillInteract
 		{
 			return TileApply(interaction as TileApply, side);
 		}
+		else if(typeof(T) == typeof(CableApply))
```

Comment:
As elsewhere - I think this should be a more generic class without item specific references. If you need a new interaction type make it as generic as possible so it can be reused, we don't want to end up with a new if statement for every possible interaction... the cable placement isn't conceptually unique.

Reply:
I' ve changed CableApply to ConnectionApply (9ef7753f67ad74949d5bfa8329a959a114742c88)